proto/regen/ecocredit/v1/state.proto

```diff
@@ -111,31 +111,35 @@ message BatchInfo {
     index : {id : 1 fields : "batch_denom", unique : true}
     index : {id : 2 fields : "project_id"}
     index : {id : 3 fields : "start_date"}
+    index : {id : 4 fields : "issuer"}
   };
 
   // id is an auto-incrementing integer to succinctly identify the batch
   uint64 id = 1;
 
+  // issuer is the address that issued the batch
+  bytes issuer = 2;
```

Comment:
For reference, this was removed during the project updates and will not require a state migration to add the field as the it currently exists in the v1alpha1 version live on mainnet and redwood.